tests/utils/testutils_lc.py

```diff
@@ -152,7 +160,8 @@ def spawn_container(self,
         container according to the image name. Return the port of container.
         """
         # Construct environment variables and start the docker container
-        worker_path = os.path.dirname(__file__)
+        PROJECT_ROOT = pathlib.Path(__file__).parent.parent.parent
```

Comment:
This might turn flaky - Add a TODO for this to be refactored.

Reply:
Added TODO